BTrees/_base.py

```diff
@@ -1269,7 +1269,7 @@ def iteritems(self, min=_marker, max=_marker,
 
     def byValue(self, min):
         return reversed(
-                sorted((v, k) for (k, v) in self.iteritems() if v >= min))
+                sorted(((v, k) for (k, v) in self.iteritems() if v >= min)))
```

Comment:
I think I was wrong about being wrong. :)

Which raises an issue.

Given that we often use reverse indexes and even given this code.....

On Python 3, a None value will error here.  I'm thinking we *should* use ``compare`` (previously ``cmp``) here.

But the inconsistency with regular Python is a little troubling.

I'm tempted to:

a) Lobby for None to be less than everything else in Python, or:

b) Create a null singleton (or maybe lownull and highnull) and migrate to it instead of None in the long term.  I wonder how the PyData folks approach this. I don't remember a formal null-like value when I was swimming in that pond a year ago. There's http://pandas.pydata.org/pandas-docs/stable/missing_data.html, which in't very hepful.